packages/perseus/src/widgets/orderer/orderer.test.ts

```diff
@@ -1,13 +1,31 @@
-import {act} from "@testing-library/react";
+import {scorePerseusItem} from "@khanacademy/perseus-score";
+import {act, screen} from "@testing-library/react";
 
 import {testDependencies} from "../../../../../testing/test-dependencies";
 import * as Dependencies from "../../dependencies";
-import {scorePerseusItemTesting} from "../../util/test-utils";
```

Comment:
Now that the scorers don't need strings, I feel like we should just get rid of `scorePerseusItemTesting`. Kind of pointless now, so good impulse to just use `scorePerseusItem`.